charts/vald/templates/index/job/correction/configmap.yaml

```diff
@@ -56,7 +56,7 @@ data:
       discoverer:
         duration: {{ $corrector.discoverer.duration }}
         client:
-          {{- $discovererClient := $index.indexer.discoverer.client }}
+          {{- $discovererClient := $index.corrector.discoverer.client }}
```

Comment:
> **NOTE**
It might be better to use a configuration defined for the corrector component.  🤔